specification/netapp/resource-manager/Microsoft.NetApp/stable/2021-06-01/netapp.json

```diff
@@ -3040,6 +3143,42 @@
         }
       }
     },
+    "SubscriptionQuotaItemList": {
```

Comment:
>SubscriptionQuotaItemList [](http://example.com/codeflow?start=5&length=25)

In case pagination is necessary, you'll need to add nextLink property, see: [Incorrect collection response](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html)